Lib/argparse.py

```diff
@@ -325,11 +325,19 @@ def _format_usage(self, usage, actions, groups, prefix):
             if len(prefix) + len(usage) > text_width:
 
                 # break usage into wrappable parts
-                part_regexp = r'\(.*?\)+|\[.*?\]+|\S+'
+                part_regexp = '|'.join([
+                    r'\(.*?\)+\s',
```

Comment:
Given my comment on the strip calls below, it would be desirable to use a lookahead assertion here so you can require trailing whitespace (or the end of the string) without capturing it in the result. Something like:

    r'\(.*?\)+(?=\s|$)'

(Note: I haven't tested that specific regex, but it's hopefully close enough to be a useful starting point)